nixos/modules/services/networking/nomad.nix

```
@@ -49,6 +49,18 @@ in
         '';
       };
 
+      settingsFiles = mkOption {
+        type = types.listOf (types.oneOf [ types.path types.str ]);
+        default = [];
+        description = ''
+          Additional settings files used to configure nomad. These files
+          will be watched for changes.
+        '';
+        example = literalExample ''
+          [ "/etc/awesome.nomad.json" ]
```

This could be more informative. Not sure which is more realistic, but we can add both.

```suggestion
          [ "/etc/nomad-mutable.json" "/run/keys/nomad-with-secrets.json" ]
```